### ISSUE TITLE
Videos play then freeze for a while, then transition to a new one and plays, then freezes and continues. 

### ISSUE THREAD

**Issue by Neccros** [OrangeJedi/Aerial#141]
Basically on either a single monitor setup or dual monitor setup the issue is the same. 

Aerial starts and plays fine then after a few min it freezes. After a few more minutes, it will transition to a new video and repeats the process.  

System specs are a Intel 11900K, 32 gigs of Corsair ram, RTX 3080 GPU

**Comment by OrangeJedi** [comment]
Hmm. Are you using the .scr patch by chance? (If that doesn't make sense, don't worry about it) If not, perhaps you could using the settings window to open the preview window. Use the keyboard shortcut `control+shift+i` to open the dev console. If it stutters can you send me a screenshot of what appears in it?

**Comment by Neccros** [comment]
Yeah I just installed it and did nothing extra... I will check in a while and will post my findings...
Thanks!

**Comment by Neccros** [comment]
I tried opening the dev console and nothing happens or opens... When/where does this combo need to be pressed?

**Comment by Neccros** [comment]
Yeah this is still an issue in Nov 2024.  I removed Aerial hoping it would be updated and recently installed it and same issue.  

Do others have this issue????
